docs/source/usage/workflows/particleFilters.rst

```diff
@@ -0,0 +1,110 @@
+.. _usage-workflows-particleFilters:
+
+Particle Filters
+----------------
+
+.. sectionauthor:: Axel Huebl
+
+A common task in both modeling, initializing and in situ processing (output) is the selection of particles of a particle species by attributes.
+PIConGPU implements such selections as *particle filters*.
+
+Particle filters are simple mappings assigning each particle of a species either ``true`` or ``false`` (ignore / filter out).
+These filters can be defined in :ref:`particleFilters.param <usage-params-core>`.
```

Comment:
QUESTION:
Is `<usage-params-core>` correct here?

Reply:
Yes.